javascript/modules/client/src/context.ts

```
@@ -140,6 +144,17 @@ export class Context {
     }
   }
 
+  useIpfsNodes(nodes: Options[]): IPFSHTTPClient[] {
+    if (nodes.length > 0) {
```

Should we accept a 0 length array? I'd throw an error

---

I think so, the library have default values if not specified pointing to localhost, so we can keep this behavior, or force the user to always send a configuration, so this array will never be 0

---

I would not expect `localhost` to be available. If there is an empty array, I would throw that error